builder/main.py

```diff
@@ -304,17 +304,17 @@ def __fetch_spiffs_size(target, source, env):
 
 elif upload_protocol in debug_tools:
     openocd_dir = platform.get_package_dir("tool-openocd-esp32") or ""
-    uploader_flags = ["-s", openocd_dir]
+    uploader_flags = ["-s", openocd_dir.replace('\\', '/')]
     uploader_flags.extend(
         debug_tools.get(upload_protocol).get("server").get("arguments", []))
     uploader_flags.extend(["-c", 'program_esp32 "{{$SOURCE}}" 0x10000 verify'])
     for image in env.get("FLASH_EXTRA_IMAGES", []):
         uploader_flags.extend(
-            ["-c", 'program_esp32 "%s" %s verify' % (image[1], image[0])])
+            ["-c", 'program_esp32 "%s" %s verify' % (image[1].replace('\\', '/'), image[0].replace('\\', '/'))])
```

Comment:
It seems that we need to modify just this line. Could you confirm?

Reply:
when I tested I needed both updates.

Reply:
all three sorry...